bin.src/ci_imsim_run.py

```diff
@@ -118,6 +121,40 @@ def run(self, currentState: BuildState):
         subprocess.run(pipetask, check=True)
 
 
+@ciRunner.register("qgraph_faro", index_command := index_command + 1)
+class QgraphFaroCommand(BaseCommand):
+    def run(self, currentState: BuildState):
+        args = (
+            "qgraph",
+            "-d", "skymap='discrete/ci_imsim/4k' AND tract=0 AND patch=24",
+            "-b", self.runner.RunDir,
+            "--input", COLLECTION,
+            "--output", COLLECTION_FARO,
+            "-p", os.path.join(os.environ["FARO_DIR"], "pipelines", "metrics_pipeline.yaml"),
+            "--skip-existing",
+            "--save-qgraph", os.path.join(self.runner.RunDir, QGRAPH_FARO_FILE),
+        )
+        pipetask = self.runner.getExecutableCmd("CTRL_MPEXEC_DIR", "pipetask", args)
```

Comment:
Can you please add `--long-log` to the pipetask call -- ci should absolutely be using it so it's easier to see what happens in a failure.

@natelust it would be nice if we could somehow have a global setting for pipetask log configuration so we don't need everyone to remember to add `--long-log`.